### PR TITLE
update to chef 16

### PR DESCRIPTION
**Pull request by TheREK3R**

### CODE REVIEW

File: libraries/resource_database.rb [hunk 1]
```diff
@@ -22,6 +22,8 @@ class Chef
   class Resource
     class Database < Chef::Resource
       resource_name :database
+      provides :database
+      provides :database
```
Comment (EMH333) [inline libraries/resource_database.rb:26]: `provides` twice?

Comment (TheREK3R) [reply libraries/resource_database.rb:26]: oops

**Review by ramereth — APPROVED** [review]
Please do a minor bump.

**Comment by TheREK3R** [comment]
!bump minor *

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a minor-level version bump to v2.7.0.  Have a nice day!

**Pull request merged**